index.js

```diff
@@ -79,9 +79,46 @@ function AtImport(options) {
           }
 
           if (stmt.type === "import") {
-            stmt.node.params = `${stmt.fullUri} ${stmt.media.join(", ")}`
+            const media = stmt.media.join(", ")
+            if (stmt.layer.length) {
+              const layerName = stmt.layer
+                .filter(layer => layer !== "")
+                .join(".")
+
+              let layerParams = "layer"
+              if (layerName) {
+                layerParams = `layer(${layerName})`
+              }
+
+              stmt.node.params = `${stmt.fullUri} ${layerParams})${media}`
```

Comment:
Why the closing parenthesis after `layerParams`?

Reply:
This is the same pattern as we see here : https://github.com/postcss/postcss-import/blob/master/test/fixtures/media-content.expected.css#L3

The `at rules` seem to be closed after each file import.
This is existing behaviour.

Reply:
This is a typo and after looking into this it is also unreachable code.
I've removed it.

Good catch!